README.md

```diff
@@ -412,5 +408,7 @@ Also, there is [`kstat`](https://openzfsonosx.org/wiki/Windows_kstat) to dynamic
 
 # Nightly builds
 
-We do not have nightly builds.  
-However, from time to time, we do quick builds for people to test a fix. [Get it here](https://openzfsonosx.org/wiki/Windows_builds).
+There are nightly builds available at [AppVeyor](https://ci.appveyor.com/project/lundman/zfsin/branch/master/artifacts)  
+- These builds are currently not signed and therefore require test mode to be enabled.
+
+There also are some historical test builds [available here](https://openzfsonosx.org/wiki/Windows_builds).
```

Comment:
I will most likely keep producing signed builds for the testers to try out - unless there is a way to do the automatic build with certificate.

Reply:
Fix pushed.